docs/changelog.rst

```diff
@@ -4,6 +4,41 @@ CHANGELOG
 Raster Vision 0.13
 -------------------
 
+This release presents a major jump in Raster Vision's power and flexibility. The most significant changes are:
```

Comment:
Good start for a blog post

Reply:
Another big feature in my opinion is the ability to skip the analyze step and train directly from uint16 rasters.